src/main/java/org/dataconservancy/packaging/shared/IpmPackager.java

```diff
@@ -0,0 +1,173 @@
+/*
```

Comment:
Checkstyle checks, even though this classes content is not totally novel (copied from elsewhere), it is "new" in this project.